posthog/api/decide.py

```diff
@@ -248,7 +248,7 @@ def get_decide(request: HttpRequest):
                 else False
             )
 
-            if str(team.id) not in (settings.NEW_ANALYTICS_CAPTURE_EXCLUDED_TEAM_IDS or []):
+            if not settings.DEBUG and str(team.id) not in (settings.NEW_ANALYTICS_CAPTURE_EXCLUDED_TEAM_IDS or []):
```

Comment:
Can you add a comment please for this

Reply:
> You could also change the dockerfile to actually expose capture's 3000,

@oliverb123 we should totally fix this, but i'm multiple distractions deep already 🤣